.github/workflows/kubernetes_test.yaml

```
@@ -155,15 +155,15 @@ jobs:
         run: |
           export JUPYTERHUB_USERNAME=${CYPRESS_EXAMPLE_USER_NAME}
           export JUPYTERHUB_PASSWORD=${CYPRESS_EXAMPLE_USER_PASSWORD}
-          jhubctl --verbose run --hub=https://github-actions.qhub.dev \
+          jhubctl --verbose run --hub=https://github-actions.nebari.dev \
                                 --auth-type=keycloak \
                                 --validate --no-verify-ssl \
                                 --kernel python3 \
                                 --stop-server \
                                 --notebook tests_deployment/assets/notebook/simple.ipynb \
 
       ### CLEANUP AFTER TESTS
-      - name: Cleanup qhub deployment
+      - name: Cleanup nebari deployment
         run: |
           cd local-deployment
-          qhub destroy --config qhub-config.yaml
+          nebari destroy --config nebari-config.yaml --disable-prompt
```

```suggestion
		  # need to bypass user confirmation prompt
          nebari destroy --config nebari-config.yaml --disable-prompt
```

added a comment for clarity